logging/api/LoggingTest/LoggingTest.csproj

```diff
@@ -8,10 +8,10 @@
 
   <ItemGroup>
     <PackageReference Include="Google.Cloud.Logging.V2" Version="2.1.0" />
+    <PackageReference Include="JUnitTestLogger" Version="0.6.0" />
```

Comment:
What is the difference between XUnit and JUnit? Is it what enables sponge to capture logs correctly?

Reply:
Sponge expects https://en.wikipedia.org/wiki/XUnit, more specifically http://nose.readthedocs.io/en/latest/plugins/xunit.html.  JUnit output conforms to that specification.

It's a horribly overloaded name.  Anyway, the xunit output format is actually a csv by default.